src/janus-api/action-service.js

```diff
@@ -155,5 +155,13 @@ export const createActionService = (
     feed.setEnabledChannel(type, boolval);
   };
 
+  that.enableVideoSubscriptions = function() {
+    feedsService.remoteFeeds().forEach((f) => f.setVideoSubscription(true));
```

Comment:
Probably. for the thumbnail mode only not local screen remote feeds should be enabled/disbaled, not all of them. 

What do you think?